### PR TITLE
Migrate terra application

### LINKED ISSUE #56: Migrate terra-application into repo
https://github.com/cerner/terra-application

### PR DESCRIPTION
**Pull request by bjankord**
Resolves #56

### CODE REVIEW

File: packages/terra-site/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "react-docgen": "^2.13.0",
     "sass-loader": "^4.1.1",
     "style-loader": "^0.13.2",
-    "terra-application": "^0.2.0",
+    "terra-application": "0.3.0",
```
Comment (Matt-Butler) [inline packages/terra-site/package.json]: good catch

File: packages/terra-application/src/Application.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@import '~terra-mixins';
```
Comment (Matt-Butler) [inline packages/terra-application/src/Application.scss]: terra-application doesn't currently have a dependency on terra-mixins. Should we remove this?

File: packages/terra-application/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
```
Comment (Matt-Butler) [inline packages/terra-application/package.json]: Also, since this doesn't have any js, do we want to remove these js dependencies?

Comment (bjankord) [inline packages/terra-application/package.json]: [Updated](https://github.com/cerner/terra-ui/pull/89/commits/710b6e091f8448bfe11954361179d41775612014)

File: packages/terra-application/src/Application.scss (hunk 2 above)
Comment (bjankord) [inline packages/terra-application/src/Application.scss]: [Updated](https://github.com/cerner/terra-ui/pull/89/commits/710b6e091f8448bfe11954361179d41775612014)

**Comment by Matt-Butler** [comment]
+1

**Pull request merged**